Runtime/Scripts/RGBotServerListener.cs

```diff
@@ -57,6 +60,7 @@ public void Awake()
             // keep this thing alive across scenes
             DontDestroyOnLoad(this.gameObject);
             _this = this;
+            _dataCollection = new RGDataCollection(this.gameObject);
```

Comment:
`RGDataCollection` not being a scriptableObject or MonoBehaviour made me do a double take as to why this was happening in `Awake`.  I would suggest moving this to `Start` (Awake can be called multiple times in a single game run instance) or just initializing it on the field itself which would be cleaner

Reply:
Moved to start - also, it now doesn't even need a reference to the gameobject.

Reply:
initializing it on the field itself would have been cleaner, but closing this comment